content/en/docs/examples/microservices-istio/run-bookinfo-with-kubernetes/index.md

```diff
@@ -0,0 +1,200 @@
+    $ kubectl get pods
+    NAME                            READY   STATUS    RESTARTS   AGE
+    details-v1-6d86fd9949-q8rrf     1/1     Running   0          10s
+    productpage-v1-c9965499-tjdjx   1/1     Running   0          8s
+    ratings-v1-7bf577cb77-pq9kg     1/1     Running   0          9s
+    reviews-v1-77c65dc5c6-kjvxs     1/1     Running   0          9s
+    {{< /text >}}
+
+1.  After the four services achieve the `Running` status, scale the deployments: let each version of each microservice run in three pods.
+
+    {{< text bash >}}
+    $ kubectl scale deployments --all --replicas 3
+    deployment "details-v1" scaled
+    deployment "productpage-v1" scaled
+    deployment "ratings-v1" scaled
+    deployment "reviews-v1" scaled
+    deployment "reviews-v2" scaled
+    deployment "reviews-v3" scaled
+    {{< /text >}}
+
+1.  Check the pods status. Notice that each microservice has three pods:
+
+    {{< text bash >}}
+    $ kubectl get pods
+    NAME                            READY   STATUS    RESTARTS   AGE
+    details-v1-6d86fd9949-fr59p     1/1     Running   0          50s
+    details-v1-6d86fd9949-mksv7     1/1     Running   0          50s
+    details-v1-6d86fd9949-q8rrf     1/1     Running   0          1m
+    productpage-v1-c9965499-hwhcn   1/1     Running   0          50s
+    productpage-v1-c9965499-nccwq   1/1     Running   0          50s
+    productpage-v1-c9965499-tjdjx   1/1     Running   0          1m
+    ratings-v1-7bf577cb77-cbdsg     1/1     Running   0          50s
+    ratings-v1-7bf577cb77-cz6jm     1/1     Running   0          50s
+    ratings-v1-7bf577cb77-pq9kg     1/1     Running   0          1m
+    reviews-v1-77c65dc5c6-5wt8g     1/1     Running   0          49s
+    reviews-v1-77c65dc5c6-kjvxs     1/1     Running   0          1m
+    reviews-v1-77c65dc5c6-r55tl     1/1     Running   0          49s
+    {{< /text >}}
+
+1.  Deploy a testing pod, [sleep]({{< github_tree >}}/samples/sleep), to use it for sending
+    requests to your microservices:
+
+    {{< text bash >}}
+    $ kubectl apply -f {{< github_file >}}/samples/sleep/sleep.yaml
+    {{< /text >}}
+
+1.  To confirm that the Bookinfo application is running, send a request to it by a curl command from your testing pod:
+
+    {{< text bash >}}
+    $ kubectl exec -it $(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}') -c sleep -- curl productpage:9080/productpage | grep -o "<title>.*</title>"
+    <title>Simple Bookstore App</title>
+    {{< /text >}}
+
+## Enable external access to the application
+
+Once your application is running, enable external (by clients from outside of the cluster) access to it. Once you
+configure the steps below successfully, you will be able to access the application by browser from your laptop.
+
+### Configure Ingress and access your application's webpage
+
+1.  Create Kubernetes Ingress:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: extensions/v1beta1
```

Comment:
Why are we using Ingress instead of Istio Gateway?

Reply:
This tutorial starts with k8s ingress and then the next modules will show you how to convert them to istio ingress, although we're getting feedback that maybe we need to use a virtualservice or gateway instead?

Reply:
The next set of modules I'm adding in the next PR will cover the addition of the istio ingress.